enterprise/backend/src/metabase_enterprise/workspaces/mirroring.clj

```diff
@@ -107,21 +99,20 @@
   ;; Set the following vars, then exec try block
   ;; for cleanup use the do block
   ;;
-  (def your-db-id 2)
-  (def your-transform (t2/select-one :model/Transform :id 1))
   (def ws* {:id "ahoj2"})
-  ;; going forward we will not be passing whole toucan model around
-  (def entities-info* {:transforms [your-transform]
-                       :inputs [(t2/select-one :model/Table :db_id your-db-id)]})
-  (-> entities-info* :inputs first :db_id)
 
-  (try (mirror-entities! ws* entities-info*)
+  (def graph* (#'metabase-enterprise.workspaces.common/build-graph
+               {:transforms (t2/select-fn-vec :id [:model/Transform :id] :workspace_id nil {:order-by [:id], :limit 1})}))
```

Comment:
As we start leveraging more keys in the graph (e.g. outputs) it will become more painful to hard code these maps.

At the least we could have a commented call for building it dynamically next to any hard-coded map, to make it easier to regenerate.